### ISSUE TITLE
[UI] Navigating through filtered emails using keyboard is broken

### ISSUE THREAD

**Issue by jacekmusial** [sj26/mailcatcher#412]
1. Populate mailcatcher with emails
2. Type something in "Search messages" to filter and to have few emails 
3. Click to open and have active selection on any filtered email
4. Press key up or key down on the keyboard to navigate through filtered emails
5. Stumble upon email that is not visible in current filtered list of emails. Blue background color of selection is not visible, email subject and its content is not matching current list of filtered emails using "Search messages..." text area.

TL:DR; If you filter emails, navigating through them using keyboard works as if you would have not filtered them.

**Comment by sj26** [comment]
Oops, sorry! I've just pushed a fix to master. Thanks for the report.
